### PR TITLE
Feature/add splash screen for game initialization

### PR DESCRIPTION
**Pull request by Alvaro2112**
I added a splash screen that will show while the game is starting (as a loading screen), to handle the async functions I use semaphores. I currently left some fake async tasks as an example of how to use the semaphores for the person that will add functions there, that's why code coverage and code smells are not great, but they will eventually be removed.

### CODE REVIEW

**Review by gauthierboeshertz — CHANGES_REQUESTED** [review]
Did you delete the logout button test?

File: app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java [hunk 1]
```diff
@@ -76,15 +102,6 @@ public void askQuestionPopupIsDisplayed() {
         onView(ViewMatchers.withId(R.id.ask_question_popup)).check(matches(isDisplayed()));
     }
 
-    @Test
```
Comment (gauthierboeshertz) [inline app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:79]: There

Comment (Alvaro2112) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:79]: Yes, it is already tested in LoginTests, it was redundant.

**Review by gauthierboeshertz — APPROVED** [review]

File: app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java (hunk 1 above)
Comment (gauthierboeshertz) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:79]: But they test two different buttons no ?

**Review by Tesa27 — APPROVED** [review]

**Pull request merged**